src/docs/development/ui/navigation/deep-linking.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Deep linking
+description: Navigate to routes when the app receives a new URL
+---
+
+{{site.alert.note}}
+This feature is only available on the dev or master channel. See [Switching
+Flutter channels][switching-channels] for more information.
+{{site.alert.end}}
+
+Flutter supports deep linking on iOS and Android in the dev channel. Any link
+matching the configured scheme will be handled by the `Navigator`. By following
+these steps, your can launch and display routes via named routes (either with
+the [`routes`][routes] parameter or [`onGenerateRoute`][onGenerateRoute]), or by
+using the [Router][] widget.
```

Comment:
Given that we've publicly called the new mechanism Navigator 2, we need to be clear what we are talking about here. This needs another note.  (I can't remember exactly what you said about the naming, but we need to be clear and I haven't seen this said anywhere else.) Something along the lines of.

```
{{site.alert.secondary}}
  **Version note:** You might be familiar with [Navigator 2.0][].
  When referring to the updated navigator mechanism, we now
  call it..."
{{site.alert.end}}
```

Reply:
Will try to get an answer on this.